### PR TITLE
Accommodate intermediate files in Standard Ingest; address DDR-566

### PR DESCRIPTION
**Pull request by coblej**

### CODE REVIEW

File: config/standard_ingest.yml.sample [hunk 1]
```diff
@@ -4,6 +4,7 @@
         - Thumbs.db
         - metadata.txt
     :targets: dpc_targets
+    :intermediate_files: intermediate_files
```
Comment (dchandekstark) [inline config/standard_ingest.yml.sample]: Is this another change that requires updating the config file?

Comment (coblej) [inline config/standard_ingest.yml.sample]: Yes, if you want Standard Ingest to recognize intermediate files as such.

Comment (dchandekstark) [inline config/standard_ingest.yml.sample]: I mean do tests require it?

Comment (coblej) [inline config/standard_ingest.yml.sample]: I'll let you know.  Removed the line from my local config file and re-running test suite.  Tests written for this functionality stub it but I suppose there might be other tests that will break.  If they do, I'll fix and update PR.

Comment (coblej) [inline config/standard_ingest.yml.sample]: @dchandekstark Test suite ran successfully on my workstation without the additional line in my config/standard_ingest.yml so I'm thinking it's not required for testing.

**Pull request merged**